include/core/LayerData.hpp

```diff
@@ -0,0 +1,59 @@
+#ifndef __NGEN_LAYER_DATA__
+#define __NGEN_LAYER_DATA__
+
+#include <unordered_map>
+#include <string>
+#include <vector>
+
+namespace ngen
+{
+    /**
+     * @brief A simple structure to hold meta data realated to a computational layer
+    */
+    
+    struct LayerDescription
+    {
+        std::string name;             //> The name of the computational layer
+        std::string time_step_units;  //> The time units that time_step is expressed in
+        int id;                       //> The layer id which determins the order that this layer will be processed in
+        double time_step;             //> The time step that will be used for models in this layer
+    };
+    
+    /**
+     * @brief A class to hold a manage layer metadata for the system
```

Comment:
```suggestion
     * @brief A class to hold and manage layer metadata for the system
```
Maybe what was intended?

Reply:
you are correct